src/inversion_scripts/invert.py

```diff
@@ -226,43 +243,25 @@ def do_inversion(
     # space, so we do not need to add 1
     # xhat = 1 + ratio
     xhat = ratio.copy()
-    xhat[:bc_idx] += 1
+    xhat[:OH_idx] += 1
 
+    # update OH scale factor by perturbation
+    if prior_err_oh > 0.0:
+        xhat[OH_idx] += perturb_oh
+        print(f"xhat[OH] = {xhat[OH_idx]}")
+        
     # Posterior error covariance matrix
     S_post = np.linalg.inv(gamma * KTinvSoK + inv_Sa)
 
     # Averaging kernel matrix
     A = np.identity(n_elements) - S_post @ inv_Sa
     
     # Print some statistics
-    print("Min:", xhat[:bc_idx].min(), "Mean:", xhat[:bc_idx].mean(), "Max", xhat[:bc_idx].max())
+    print("Min:", xhat[:OH_idx].min(), "Mean:", xhat[:OH_idx].mean(), "Max", xhat[:OH_idx].max())
```

Comment:
Same as above, I think this should use `scale_factor_idx`

Reply:
Now fixed in https://github.com/geoschem/integrated_methane_inversion/pull/157/commits/67f8f8a651ba6037f3cb882049755eb82ecb32c3